gossipd/gossip.c

```diff
@@ -1509,7 +1509,10 @@ static struct io_plan *gossip_init(struct daemon_conn *master,
 	}
 	daemon->rstate = new_routing_state(daemon, &chain_hash, &daemon->id);
 
-	setup_listeners(daemon, port);
+	if (port)
+		setup_listeners(daemon, port);
+	else
+		status_trace("Not listeing for incoming connections due to --no-listen");
```

Comment:
s/listeing/listening/ and /--no-listen/--offline/.

Also, this duplicates the check inside setup_listeners; one should go...

Reply:
Will remove the outer check.

Reply:
Fewer chances for typos :wink: